components/pkg-export-kubernetes/defaults/KubernetesManifest.hbs

```diff
@@ -24,6 +24,8 @@ spec:
   ## An object containing parameters that affects how the Habitat service
   ## is executed.
   service:
+    ## Name of the Habitat service.
```

Comment:
Not sure if I wouldn't say "Name of the Habitat pkg" here. As this is what we use to form the path to place the `user.toml` file. Not sure which one would make it more clear, service or pkg.

Reply:
Judging by the rest of the code, service name sounds like the right thing to call this; package name implies (to me) at least an `origin/name` identifier, whereas we currently refer to services just by their names.

Reply:
Since I called it a `Habitat service` in the comment that @LiliC was talking about, I'm wondering if you want me to change the name of the variables/fields from `hab_name` to `service_name`?

Reply:
@zeenix Yeah, that'd probably make things more consistent and easier to keep straight.

Reply:
It's done. Sorry, i ammended the patch already so it was a forced push. Hope that's ok in this simple case. I only changed the name of the fields/variables.

Reply:
@zeenix It's OK here. 

Also, the rename makes it much clearer.